spec/std/http/response_spec.cr

```diff
@@ -44,9 +44,10 @@ module HTTP
       response.body?.should be_nil
     end
 
-    it "parses response with duplicated headers" do
+    it "parses response with cookies" do
       response = Response.from_io(StringIO.new("HTTP/1.1 200 OK\r\nContent-Type: text/plain\r\nContent-Length: 5\r\nSet-Cookie: a=b\r\nSet-Cookie: c=d\r\n\r\nhelloworld"))
-      response.headers.get("Set-Cookie").should eq(["a=b", "c=d"])
+      response.cookies["a"].value.should eq("b")
+      response.cookies["c"].value.should eq("d")
     end
```

Comment:
This is true, i'm not sure which headers should be used instead to test this behavior though as Set-Cookie is the only one I know that occurs more than once.